.github/workflows/trigger_staging_deploy.yml

```diff
@@ -0,0 +1,38 @@
+name: Trigger release to staging
+
+on:
+  workflow_dispatch:
+    inputs:
+      version:
+        description: Which version are you creating?
+        required: true
+      ref:
+        description: Git ref to deploy
+        required: true
+        default: develop
+jobs:
+  create-pr:
+    runs-on: ubuntu-20.04
+    steps:
+      - uses: actions/checkout@v2
+        name: Checkout Code
+        with:
+          ref: ${{ github.event.inputs.ref }}
+
+      - name: Update master
+        run: |
+          git fetch
+          git branch tmp-branch
+          git switch master
+          git reset --hard tmp-branch --
```

Comment:
Unfortunately this doesn't mean we get to keep the original commits, as we still have the PR merge